web/src/actions/admin.js

```diff
@@ -1,45 +1,19 @@
-import { notify } from './notification';
```

Comment:
I'm hopeful we'll eventually get rid of the toast/snackbar entirely, but for now, we can at least ditch it from the admin alerts.

![](https://media.giphy.com/media/11ctq1pDmD3cB2/giphy.gif)